Tests/Request/ParamFetcherTest.php

```diff
@@ -586,16 +579,12 @@ public function testConstraintThrowExceptionInStrictMode()
             ->method('read')
             ->will($this->returnValue(array('bizoo' => $param)));
 
-        $this->violationFormatter->expects($this->once())
-            ->method('formatList')
-            ->will($this->returnValue('foobar'));
-
         $this->setExpectedException(
             '\\Symfony\\Component\\HttpKernel\\Exception\\BadRequestHttpException',
-            'foobar'
+            "Parameter \"bizoo\" of value \"\" violated a constraint \"expected message 1\"\nParameter \"bizoo\" of value \"\" violated a constraint \"expected message 2\""
```

Comment:
Well with this change the ViolationFormatter is now completely useless (even if i don't know if someone ever used it).
IMO the exception message should be generated by the ViolationFormatter in this branch whereas in 2.0 where it is generated by the exception.